start.yaml

```
@@ -1,12 +1,25 @@
 cheat: 0
 health: 10
+armor protection: .95
 held item: Sword
+held chestplate: Leather Jacket
+held boots: Leather Boots
+held leggings: Old Pants
+inventory slots: 9
+inventory slots remaining: 4
 inventory:
 - Sword
 - Healing Potion
-max health: 10
+- Leather Jacket
+- Leather Boots
+- Old Pants
+- Stretched Bag
+max health: 15
 x: 0
 xp: 0
 y: 0
+map_zone: "Kathallion Plains"
```

```suggestion
map zone: "Dungeon Entrance"
```
You forgot to do this one.